deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -139,6 +141,8 @@ spec:
           args:
             - "--v=5"
             - "--endpoint=unix:/csi/csi.sock"
+            - "--supports-dynamic-iops-provisioning=hyperdisk-balanced,hyperdisk-extreme"
+            - "--supports-dynamic-throughput-provisioning=hyperdisk-balanced,hyperdisk-throughput,hyperdisk-ml"
```

Comment:
Will adding these here make manual deploy with stable-master fail? The integration tests use the dev overlay, but when customers manually deploy the driver, they will use stable-master overlay: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/dbd3bfa77602656f0a6f047997389d6d3727b93a/docs/kubernetes/user-guides/driver-install.md

Which uses an image that won't have these tags implemented yet right?  https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/v1.14.2/deploy/kubernetes/images/stable-master/image.yaml#L55 . Are you able to try out the manually deployment of the driver to see if this works using the instructions we have [here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/dbd3bfa77602656f0a6f047997389d6d3727b93a/docs/kubernetes/user-guides/driver-install.md)

Reply:
Resolved offline